docs/data/indexers/build-your-own/README.mdx

```diff
@@ -8,7 +8,3 @@ Learn about the services and tools that you can build custom indexing with.
 ## [Galexie](./galexie/README.mdx)
 
 Galexie is a tool for extracting, processing, exporting Stellar ledger metadata to external storage, and creating a data lake of pre-processed ledger metadata. Galexie is the foundation of the Composable Data Pipeline (CDP) and serves as the first step in extracting raw Stellar ledger metadata and making it accessible. Learn more about CDP’s benefits and applications in [this blog post](https://stellar.org/blog/developers/composable-data-platform).
-
-## Ingest Library
-
-Placeholder for ingest docs
```

Comment:
FYI @urvisavla @sreuland this is what I was referencing over here - https://github.com/stellar/stellar-docs/pull/1415#issuecomment-2721658677

Reply:
ok, we didn't catch this TODO on feature branch yet - http://developers-pr1346.previews.kube001.services.stellar-ops.com/ , i can add a brief summary for ingest library as an indexer tool, on the feature branch pr.